Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -704,18 +704,24 @@ def homebrew
       return if @skip_homebrew
 
       if @tap.nil? && Array(@formulae).empty?
-        tests_args = ["--official-cmd-taps"]
-        tests_args_no_compat = []
-        tests_args_no_compat << "--coverage" if ARGV.include?("--coverage")
-        test "brew", "tests", *tests_args
-        test "brew", "tests", "--generic", *tests_args
-        test "brew", "tests", "--no-compat", *tests_args_no_compat
+        coverage_args = []
+        if ARGV.include?("--coverage")
+          if ENV["JENKINS_HOME"]
+            if OS.mac? && MacOS.version == :el_capitan
+              coverage_args << "--coverage"
+            end
+          else
+            coverage_args << "--coverage"
+          end
+        end
```

Comment:
Is `test-bot` called with `--coverage` on Jenkins? If not, the `if ENV["JENKINS_HOME"]` check should probably be on the same level as `if ARGV.include?("--coverage")`.